### ISSUE TITLE
na_ontap_export_policy_rule doesn't accept user name for option "anonymous_user_id"

### ISSUE THREAD

**Issue by mamoep** [ansible-collections/netapp.ontap#5]
##### SUMMARY

na_ontap_export_policy_rule has the option anonymous_user_id which is documented as "User name or ID to which anonymous users are mapped". Type is int so user name is not accepted, only IDs. SSH command "export-policy rule create" accepts user name or ID.

##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME

na_ontap_export_policy_rule

##### ANSIBLE VERSION

```paste below
2.9.1.9
```

##### CONFIGURATION

```paste below

```

##### STEPS TO REPRODUCE

```yaml
  - name: Create export policy rules
    na_ontap_export_policy_rule:
      state: present
      name: "{{ export_policy }}"
      vserver: "{{ vserver_name }}"
      client_match: "{{ rule.clientmatch }}"
      ro_rule: "{{ rule.rorule }}"
      rw_rule: "{{ rule.rwrule }}"
      protocol: nfs
      super_user_security: "{{ rule.superuser }}"
      anonymous_user_id: username
      <<: *login
    connection: local

```

##### EXPECTED RESULTS

Export rule should be added.

##### ACTUAL RESULTS

Task fails

```paste below
"msg": "argument anonymous_user_id is of type <type 'str'> and we were unable to convert to int: <type 'str'> cannot be converted to an int"
```

**Comment by lonico** [comment]
Yes, seems it is a misunderstanding because the ZAPI field is identified as anonymous-user-id

We'll change it to str.

**Comment by lonico** [comment]
fixed in 7eb3d659bb7612122bd3e3ca9ca1793c4a694a0c

will be released as 21.8.0 in Galaxy.

**Comment by lonico** [comment]
fixed in 21.8.0
